### PR TITLE
feat: use new security pocily

### PR DESCRIPTION
**Pull request by olenagerasimova**
part of #145 

used new auth slices and mechanism in the adapter.

The PR is failing as local `http` version is used here. Also, please, note, that now we have to pass repository name to the adapter (this one and any other in the future). Any ideas to implement it better?

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@dgarus ready for review

File: src/main/java/com/artipie/files/FilesSlice.java [hunk 1]
```diff
@@ -68,21 +70,25 @@ public final class FilesSlice extends Slice.Wrap {
      * @param storage The storage. And default parameters for free access.
      */
     public FilesSlice(final Storage storage) {
-        this(storage, Permissions.FREE, Authentication.ANONYMOUS);
+        this(storage, Policy.FREE, Authentication.ANONYMOUS, "repo-name");
```
Comment (dgarus) [inline src/main/java/com/artipie/files/FilesSlice.java:73]: Maybe create
```
static final String ANY_REPO = "*";
```
and use it anywhere we don't need a certain repo name?

Comment (olenagerasimova) [reply src/main/java/com/artipie/files/FilesSlice.java]: added

**Review by dgarus — APPROVED** [review]
lgtm

**Pull request merged**